apps/finance/app/package.json

```diff
@@ -46,11 +46,15 @@
   },
   "scripts": {
     "lint": "eslint ./src",
-    "start": "npm run sync-assets && npm run watch:script & parcel serve index.html -p 3002 --out-dir build/",
-    "build": "npm run sync-assets && npm run build:script && parcel build index.html --no-cache --out-dir build/ --public-url \".\"",
-    "build:script": "parcel build src/script.js --out-dir build/",
-    "watch:script": "parcel watch src/script.js --out-dir build/ --no-hmr",
-    "sync-assets": "copy-aragon-ui-assets -n aragon-ui ./build && rsync -rtu ./public/ ./build",
+    "test": "jest",
+    "start": "npm run sync-assets && npm run watch:script & parcel serve index.html -p 3002 --out-dir ../dist/",
+    "watch": "npm run watch:script",
```

Comment:
Just wondering, do we need this for buidler?

Reply:
Yes, the scripts `serve`, `watch` and `sync-assets` are pre-requisites of the plugin `start` task if there is an app directory. 

I think this is a harsh restriction for the user at the moment. Maybe in a future version, we should give configurations to specify the scripts and don't make `sync-assets` a must-have.

Reply:
Definitely—we should also double check if they exist and warn if they don't rather than make it a hard requirement.